<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIClass</class>
 <widget class="QWidget" name="UIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI</string>
  </property>
  <widget class="QTextEdit" name="cli">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plot">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Commands</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clr">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections/>
</ui>
